src/subcommand/wallet/pending.rs

```diff
@@ -0,0 +1,20 @@
+use {super::*, crate::wallet::entry::EtchingEntry};
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Clone)]
+pub struct PendingOutput {
+  pub etchings: Vec<EtchingEntry>,
+}
```

Comment:
```suggestion
pub struct PendingOutput {
  pub rune: SpacedRune,
  pub commit: Txid,
}
```

Let's just return the Rune name and the commit Txid for now.